.buildkite/pr_main_mr_test.yml

```diff
@@ -1,7 +1,7 @@
 steps:
   - label: "Execute tests"
     command: ".buildkite/scripts/test.sh"
-    branches: "master"
+    if: build.branch == "master" || build.env("BUILDKITE_PULL_REQUEST") != "false"
```

Comment:
probably yes, but in that case it will starts when somebody starts it manually, if we speak only about triggers to merge or pr it will works, but up to you

Reply:
it'd work for manual, branches and PR - which I think is fine

Reply:
okay, deleted

Reply:
IMO, we should run the tests:
- On PRs, prior to when they get merged
- In the master branch, on every push to the branch (including PR merges)

I don't see a need to run tests on other branches unless/until there's a PR opened for them.

Reply:
@TrushynAliaksandr my bad - your original approach was the good one - so sorry.